.github/taskdef/prod-zkevm-testnet.yaml

```diff
@@ -2,8 +2,8 @@ region: eu-west-1
 hostport: 80
 containerport: 80
 role: backend-apps
-environment: staging
-iac: terraform-workspace-aws-test-apps-eu-west-1-apps-auto-claim-staging-polygon-technology
+environment: prod
+iac: terraform-workspace-aws-test-apps-eu-west-1-apps-zkevm-testnet-auto-claim-polygon-technology
```

Comment:
Should be terraform-workspace-aws-prod-applications-eu-west-1-apps-auto-claim-zkevm-testnet-auto-claim